### ISSUE TITLE
switch off sanity swaps (as default)

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#970]
Description below: https://github.com/ExchangeUnion/xud/issues/970#issuecomment-500808616

**Comment by sangaman** [comment]
Thinking about the best approach here, we'll need to track the unbroadcasted orders somehow. I'm thinking we could use another fast priority queue, same as the order book, except ordered by quantity smallest on top. When the capacity increases, we start checking the top item if it's less than the capacity. If so, we broadcast it, remove it from the queue, and repeat for the next item in the list. If not we stop and wait until the next capacity increase. Any unbroadcast orders get added to the queue in real time.

**Comment by kilrau** [comment]
As agreed on the call, we go for temporarily making the checks non-blocking with warnings as log entries and CLI outputs first. Opened https://github.com/ExchangeUnion/xud/issues/971 for better handling and making them blocking again.

**Comment by kilrau** [comment]
And more:
* actual sanity swaps with preimage release should only happen with direct channel peers
* for multi-hop peers: setup htlcs, but don't release preimage. Swap partner has to be aware that this is the sanity swap test to not punish us in future for this behavior. Reason: routing fees

**Comment by kilrau** [comment]
Moved sanity swap fixes to https://github.com/ExchangeUnion/xud/issues/988, this is only about making them non-blocking.

**Comment by kilrau** [comment]
Decision from the call: just switch off sanity swaps on default, making them non-blocking in #988

**Comment by kilrau** [comment]
I just ran into a couple of situations on simnet (where `sanitychecks=false`), where I entered an order without having a channel or sufficient channel balance. The balance checks on `placeorder` would have caught that. And as far as I know, there are no problems with these. So my strong suggestion is to do the following:
* separate config option in the category `#sanity checks` with defaults `sanityswaps=false`, `balancechecks=true`.
* This leaves the balance checks on placeorder enabled and will prevent *many* bug reports
